packages/@aws-cdk/aws-servicecatalog/README2.md

```diff
@@ -0,0 +1,379 @@
+enables organizations to create and manage catalogs of products for their end users that are approved for use on AWS.
+
+## Table Of Contents
+
+- [Portfolio](#portfolio)
+  - [Granting access to a portfolio](#granting-access-to-a-portfolio)
+  - [Sharing a portfolio with another AWS account](#sharing-a-portfolio-with-another-aws-account)
+- [Product](#product)
+  - [Creating a product from a local asset](#creating-a-product-from-local-asset)
+  - [Creating a product from a stack](#creating-a-product-from-a-stack)
+  - [Adding a product to a portfolio](#adding-a-product-to-a-portfolio)
+- [TagOptions](#tag-options)
+- [Constraints](#constraints)
+  - [Tag update constraint](#tag-update-constraint)
+  - [Notify on stack events](#notify-on-stack-events)
+  - [CloudFormation template parameters constraint](#cloudformation-template-parameters-constraint)
+  - [Set launch role](#set-launch-role)
+  - [Deploy with StackSets](#deploy-with-stacksets)
+
+
+The `@aws-cdk/aws-servicecatalog` package contains resources that enable users to automate governance and management of their AWS resources at scale.
+
+```ts nofixture
+import * as servicecatalog from '@aws-cdk/aws-servicecatalog';
+```
+
+## Portfolio
+
+AWS Service Catalog portfolios allow administrators to organize, manage, and distribute cloud resources for their end users.
+Using the CDK, a new portfolio can be created with the `Portfolio` construct:
+
+```ts
+new servicecatalog.Portfolio(this, 'Portfolio', {
+  displayName: 'MyPortfolio',
+  providerName: 'MyTeam',
+});
+```
+
+You can also specify optional metadata properties such as `description` and `messageLanguage`
+to help better catalog and manage your portfolios.
+
+```ts
+new servicecatalog.Portfolio(this, 'Portfolio', {
+  displayName: 'MyFirstPortfolio',
+  providerName: 'SCAdmin',
+  description: 'Portfolio for a project',
+  messageLanguage: servicecatalog.MessageLanguage.EN,
+});
+```
+
+Read more at [Creating and Managing Portfolios](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/catalogs_portfolios.html).
+
+To import an existing portfolio into your CDK application, use the `Portfolio.fromPortfolioArn()` factory method:
+
+```ts
+const portfolio = servicecatalog.Portfolio.fromPortfolioArn(this, 'ImportedPortfolio',
+  'arn:aws:catalog:region:account-id:portfolio/port-abcdefghi');
+```
+
+### Granting access to a portfolio
+
+You can grant access to and manage the `IAM` users, groups, or roles that have access to the products within a portfolio. 
+Entities with granted access will be able to utilize the portfolios resources and products via the console or AWS CLI.
+Once resources are deployed end users will be able to access them via the console or service catalog CLI.
+
+```ts fixture=basic-portfolio
+import * as iam from '@aws-cdk/aws-iam';
+
+const user = new iam.User(this, 'User');
+portfolio.giveAccessToUser(user);
```

Comment:
What does "giveAccess" mean? Usually we call this `grantXXX` where XXX stands for the actions the identity will be able to perform.

For example: `grantDeploy` or `grantConsume` or `grantRead`. 

Otherwise perhaps `grantAccess`, though I'd prefer a verb.

Reply:
make default to attach base level servicecatalog api calls.